### PR TITLE
Update Package Names for Fedora

### PR DESCRIPTION
**Pull request by abe-101**
Following the installation guide to install scipy system wide on fedora:
`sudo dnf install numpy scipy python-matplotlib ipython python-pandas sympy python-nose atlas-devel`
Produces the error:
```
No match for argument: scipy
No match for argument: sympy
Error: Unable to find a match: scipy sympy
```
I've updated the guide to include the correct package names:
`sudo dnf install python-numpy python-scipy python-matplotlib ipython python-pandas python-sympy python-nose atlas-devel`

### CODE REVIEW

**Review by tupui — APPROVED** [review]
Thank you @abe-101. LGTM but I am not a Fedora user.

@jarrodmillman I think you use Fedora right? Anything else to do here?

**Comment by tupui** [comment]
Here https://developer.fedoraproject.org/tech/languages/python/scipy.html I can see that they use `python3-...` instead. Not sure what's preferred.

**Comment by abe-101** [comment]
> Here https://developer.fedoraproject.org/tech/languages/python/scipy.html I can see that they use `python3-...` instead. Not sure what's preferred.
> 
> Also I thought we cleaned this page a bit more than that. We are still referring to other packages than SciPy. cc @melissawm

It seems that `python-...` will point to the version of python the user is running. I'm not sure what the preferred way is.
Based on a chat I had with other fedora users it seems `python3-...` would be "more correct".
I've gone ahead and updated it.
Thank you @tupui for pointing it out.

**Comment by jarrodmillman** [comment]
I would use `python3-ipython` as well.  We don't need `python3-nose` or `atlas-devel`.

The whole page needs some attention, but that can be in a separate PR.

File: content/en/install.md [hunk 1]
```diff
@@ -90,7 +90,7 @@ using apt-get:
 
 using dnf:
 
-    sudo dnf install numpy scipy python-matplotlib ipython python-pandas sympy python-nose atlas-devel
+    sudo dnf install python3-numpy python3-scipy python3-matplotlib ipython python3-pandas python3-sympy python3-nose atlas-devel
```
Comment (jarrodmillman) [inline content/en/install.md:93]:
```suggestion
    sudo dnf install python3-numpy python3-scipy python3-matplotlib python3-ipython python3-pandas python3-sympy python3-pytest
```

**Review by jarrodmillman — CHANGES_REQUESTED** [review]

**Comment by jarrodmillman** [comment]
I would also change the heading from `Fedora 22 and later` to `Fedora`.

**Review by jarrodmillman — APPROVED** [review]

**Comment by jarrodmillman** [comment]
Thanks!

**Pull request merged**